### PR TITLE
fix(surveys): fix survey template

### PR DESCRIPTION
**Pull request by liyiy**
## Problem

Appearance wasn't being applied from the survey template

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/__snapshots__/scenes-app-surveys--survey-templates.png
Comment (neilkakkar) [inline frontend/__snapshots__/scenes-app-surveys--survey-templates.png:1]: nit: can we change emoji colours, with the background, they now look too dull?

**Review by neilkakkar — APPROVED** [review]

File: frontend/__snapshots__/scenes-app-surveys--survey-templates.png
Comment (liyiy) [reply frontend/__snapshots__/scenes-app-surveys--survey-templates.png:1]: The rating buttons are white by default but on emoji surveys, it looks super washed, compared to the updated one

Comment (liyiy) [reply frontend/__snapshots__/scenes-app-surveys--survey-templates.png:1]: we can change it though, any suggestions 😆

Comment (liyiy) [reply frontend/__snapshots__/scenes-app-surveys--survey-templates.png:1]: Screen Shot 2023-10-17 at 11 14 00 AM

Comment (neilkakkar) [reply frontend/__snapshots__/scenes-app-surveys--survey-templates.png:1]: oh wrong person to ask for colours, not sure if it was a snapshot issue but the ones you shared above look better!^^ Ship it!

(cc: @corywatilo for any color suggestion here)

Comment (liyiy) [reply frontend/__snapshots__/scenes-app-surveys--survey-templates.png:1]: betteeeer

**Pull request merged**